PubSub/src/V1/SubscriberGrpcClient.php

```diff
@@ -2,7 +2,7 @@
 // GENERATED CODE -- DO NOT EDIT!
 
 // Original file comments:
-// Copyright 2017 Google Inc.
+// Copyright 2018 Google Inc.
```

Comment:
We might need to make sure we run a replace here. I am actually more in favor of removing these files altogether since we aren't using them under the hood in the GAPIC. There is some value proposed, however, in case users would like to use these clients directly. WDYT?